rs/nervous_system/tools/release-runscript/src/main.rs

```diff
@@ -182,26 +209,131 @@ If not, you can also run the upgrade tests manually:
    - Example: Test at rs/nervous_system/integration_tests/tests/sns_release_qualification.rs",
     );
 
-    run_create_proposal_texts(CreateProposalTexts);
+    run_create_proposal_texts(CreateProposalTexts {
+        commit,
+        nns_canisters,
+        sns_canisters,
+    });
 }
 
-fn run_create_proposal_texts(_: CreateProposalTexts) {
+fn run_create_proposal_texts(cmd: CreateProposalTexts) {
+    let CreateProposalTexts {
+        commit,
+        nns_canisters,
+        sns_canisters,
+    } = cmd;
+
+    // Create proposals directory under ic_dir/proposals/release-{ISO_DATE}
+    let ic = ic_dir();
+    let proposals_dir = ic.join("proposals");
```

Comment:
might not want to make them in ic directory, as they'll get lost during certain git operations, or accidentally added to commits